docs/contribute.md

```diff
@@ -1,328 +1,141 @@
-# Contribute
+# Contributing to Ralph
 
-## Ralph's core
+Thank you for considering contributing to Ralph! We appreciate your interest and support. This documentation provides guidelines on how to contribute effectively to our project.
 
-To start playing with `ralph`, you should build it using the `bootstrap` Make target:
+## Issues
 
-```
-$ make bootstrap
-```
+Issues are a valuable way to contribute to Ralph. They can include bug reports, feature requests, and general questions or discussions. When creating or interacting with issues, please keep the following in mind:
 
-Once the project has been bootstrapped, you may want to edit generated `.env`
-file to set up available backends parameters that will be injected in the
-running container as environment variables to configure Ralph (see [backends
-documentation](./backends.md)):
+### 1. Search for Existing Issues
 
-```dotenv
-# Elasticsearch backend
-RALPH_ES_HOSTS=http://elasticsearch:9200
-RALPH_ES_INDEX=statements
-RALPH_ES_TEST_HOSTS=http://elasticsearch:9200
-RALPH_ES_TEST_INDEX=test-index
+Before creating a new issue, search the existing issues to see if your concern has already been raised. If you find a related issue, you can add your input or follow the discussion.
 
-# [...]
-```
+### 2. Creating a New Issue
 
-> Note that lines starting with a `#` are considered as commented and thus will
-> have no effect while running Ralph.
+**Bug Report**
 
-Now you can start playing the CLI:
+- Use this template if you've encountered a bug in our project.
+- Include a clear and concise title that summarizes the issue.
+- Provide detailed steps to reproduce the bug.
+- Include information about your environment (e.g., operating system, browser version, etc.).
 
-```
-$ bin/ralph --help
-```
+**Feature Request**
 
-To lint your code, either use the `lint` meta target or one of the linting tools we use:
+- Use this template if you have a feature you'd like to suggest.
+- Provide a descriptive title.
+- Explain the feature you're proposing and its potential benefits.
 
-```bash
-# Run all linters
-$ make lint
+**Support Question**
 
-# Run pylint
-$ make lint-pylint
+- Use this template if you have a question or need assistance with our project.
+- Provide a clear and concise title.
+-  Detail your question or problem.
 
-# List available linters
-$ make help | grep lint-
-```
+**Security Vulnerability**
 
-To run tests on your code, either use the `test` Make target or the
-`bin/pytest` script to pass specific arguments to the test runner:
+If you discover a security vulnerability, please report it directly to us following our Security Vulnerability Reporting Guidelines.
```

Comment:
I don't think we need to add a description of "report a bug", "security vulnerability" etc. as they are already included in the github template. A concise bullet point list would be sufficient in my opinion.